ansible/playbook.yaml

```
@@ -0,0 +1,69 @@
+- name: update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      private: false
+      prompt: What is the Trello API key
+
+    - name: trello_api_token
+      prompt: What is the Trello API token
+
+  tasks:
+  - name: Install the lastest version of Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: latest
+
+  - name: Install the lastest version of Python
+    become: yes
+    ansible.builtin.yum:
+      name: python311
+      state: latest
```

Not a suggestion as such, but worth noting that we can combine these if we want to:
```suggestion
  - name: Install the latest version of Git & Python
    become: yes
    ansible.builtin.yum:
      name:
        - git
        - python311
      state: latest
```